pkgs/development/python-modules/torchmetrics/default.nix

```diff
@@ -40,46 +41,47 @@ buildPythonPackage {
   ];
 
   # Let the user bring their own instance
-  buildInputs = [
-    torch
-  ];
-
-  nativeCheckInputs = [
-    pytorch-lightning
-    scikit-learn
-    scikit-image
-    cloudpickle
-    psutil
-    pytestCheckHook
-  ];
+  buildInputs = [ torch ];
```

Comment:
Just formatting

Reply:
?

Reply:
`buildInputs` haven't changed, only the formatting has -> extra diff

Reply:
What I meant is: might as well revert it and have a small diff to receive comments on